commands/user_e2e_test.go

```diff
@@ -136,3 +136,72 @@ func (s *MmctlE2ETestSuite) TestSearchUserCmd() {
 		s.Equal("Unable to find user '"+emailArg+"'", printer.GetErrorLines()[0])
 	})
 }
+
+func (s *MmctlE2ETestSuite) TestCreateUserCmd() {
+	s.SetupTestHelper().InitBasic()
+
+	s.RunForAllClients("Don't create new user w/o username", func(c client.Client) {
+		printer.Clean()
+		cmd := &cobra.Command{}
```

Comment:
Shall we add other required parameters here?
```suggestion
		cmd := &cobra.Command{}
		cmd.Flags().String("password", "somepass", "")
		cmd.Flags().String("email", s.th.GenerateTestEmail(), "")
```